app/src/main/java/knu/dong/teamproject/SignupActivity.kt

```diff
@@ -61,16 +78,32 @@ class SignupActivity : AppCompatActivity() {
 
             signUp(email, verifyCode, password)
         }
-    }
-    private fun signUp(email: String, verifyCode: String, password: String) {
-        // TODO: 추후 회원가입 요청 구현, 현재는 그냥 가입 완료 메시지만
-        val result = true
-        if (result) {
-            Toast.makeText(this@SignupActivity, "성공적으로 회원가입 되었습니다.", Toast.LENGTH_SHORT).show()
-            finish()
+
+        binding.titleBar.btnBack.setOnClickListener {
+            onBackPressed()
         }
-        else {
-            Toast.makeText(this@SignupActivity, "회원가입 요청이 실패했습니다.", Toast.LENGTH_SHORT).show()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        job.cancel()
+    }
+    private fun signUp(email: String, verify: String, password: String) {
+        launch(Dispatchers.Main) {
+            val result = HttpRequestHelper(this@SignupActivity)
+                .post("api/users")
+                {
+                    contentType(ContentType.Application.Json)
+                    setBody(Signup(email, verify.toInt(), password))
+                }
+            if (result?.status == HttpStatusCode.Created) {
+                Toast.makeText(this@SignupActivity, "성공적으로 회원가입 되었습니다.", Toast.LENGTH_SHORT).show()
+                finish()
+            }
+            else {
+//                Log.d("chae", "오류코드: ${result?.status}, 오류메시지: ${result?.toString()}")
```

Comment:
```suggestion
                Log.d("chae", "오류코드: ${result?.status}, 오류메시지: ${result?.toString()}")
```

Logger로 남기는 로그는 굳이 안지우고 냅둬도 됨. println같이 콘솔에 바로 출력되는 애들은 남겨두면 안 좋은데, Logger는 설정으로 쉽게 운영에서는 없앨 수 있어서 ㅎ

Reply:
반영완료